### ISSUE TITLE
[BUG] Impossible to create a Manager instance with GCP

### ISSUE THREAD

**Issue by lezan** [state-of-the-art/BlendNet#143]
### Issue description:
I installed succesfully BlenNet and I was trying to start Manager but got this issue. Waited around 10m but nothing changed.
BlenNet status is not moving from "Manager starting..." (I think it need to go into idle status).

If I open system console on Blender I can see:
```
DEBUG: Trying to get CA certificate from the bucket
INFO: Downloading data from "gs://...."
WARN: Download failed: <HttpError 404 when requesting https://storage..../o/ca.crt?alt=media returned "Not Found">
DEBUG: Periodic update manager info False
```

Check into Google console I can also see two other errors:
1. Already exist (HTTP 409): The resource 'projects/blender-304216/global/firewalls/blendnet-manager-8443' already exists.
2. Argument not valid (HTTP 400): Setting minimum CPU platform is not supported for the selected machine type e2-standard-16.

For the first one it is not a real problem, I just need to remove rule already created (maybe we need to check if rule already exist, without giving error?) or ignore it (rule is already here).
For the second one I am not sure how to solve it, maybe I missconfig something?

### Environment:
* **Application version**: 0.3.7
* **Blender client version**: 2.90.1
* **Blender worker version**: 2.90.1
* **OS**: Windows 10

### Steps to reproduce:
1. Run Blender
2. Open project
3. Install and configure BlenNet
4. Click on + to start Manager (or Run Manager instance, is the same right?)
5. Open system console and see error.

### Expected behavior
Manager correclty instanced.

### Screenshots

Error into system console:
![Immagine](https://user-images.githubusercontent.com/1663016/107363149-a8f0e200-6ad1-11eb-932e-e25c1b4acb58.png)

BlenNet config:
![Immagine2](https://user-images.githubusercontent.com/1663016/107374090-23742e80-6adf-11eb-894f-24c79d765f3d.png)

### Additional context
Please check https://github.com/state-of-the-art/BlendNet/wiki/If-you-stuck-with-an-issue to attach
some useful logs here.

**Comment by rabits** [comment]
Hi @lezan , so the first thing I need is the logs. Those parts of the logs are not helping much, so could you please attach the Addon logs: https://github.com/state-of-the-art/BlendNet/wiki/If-you-stuck-with-an-issue ?

The issue with `ca.crt` is not a big deal - it is generated only when the Manager starting, so Addon just trying to get it to be able to connect to the Manager.

The issue with minimum CPU platform is related to https://github.com/state-of-the-art/BlendNet/issues/140 - I think it's not working if you not using intel instance, so most probably I will need to remove it from the configuration, because it's just an optimization.

So could you please try to remove those lines from BlendNet addon on your local and try to run the Manager again?:
* Manager: https://github.com/state-of-the-art/BlendNet/blob/v0.3.7/BlendNet/providers/gcp/__init__.py#L367
* Agent: https://github.com/state-of-the-art/BlendNet/blob/v0.3.7/BlendNet/providers/gcp/__init__.py#L490

If it will work - I will just remove this optimization in the next 0.3.8 version and everyone will be happy)

**Comment by rabits** [comment]
Hi @lezan , did it worked for you?

**Comment by lezan** [comment]
HI @rabits and sorry to answer so late, but I got some work to do in the last days and I did not find time to try to follow your suggestion.

I already try in the first post to attach logs mentioned [here](https://github.com/state-of-the-art/BlendNet/wiki/If-you-stuck-with-an-issue) but I did not find both button (Node log and Service log) in GUI. I try to remove and reinstall several time plugin, but it did not change. What can I do?

> The issue with minimum CPU platform is related to #140 - I think it's not working if you not using intel instance, so most 
> probably` I will need to remove it from the configuration, because it's just an optimization.

So the issue born from an instance machine with AMD as CPU? If I choose an instance without AMD is fixed? What kind of optmization are we talk? Is it a big deal?

> So could you please try to remove those lines from BlendNet addon on your local and try to run the Manager again?:

Are you saying I need to download [that](https://github.com/state-of-the-art/BlendNet/releases/download/v0.3.7/blendnet-0.3.7.zip), unzip, change those lines, zip and install?

Thanks for your support and for this plugin and sorry to answer so late!

**Comment by rabits** [comment]
Oh, no worries about the node buttons - they are available only when instance can be actually created. I mean more Addon logs, which is available when you run blender in console - they are printing to the stdout.

Regarding the changes I asked for - you can actually change the BlendNet addon in the directory where it's installed (by install Blender just unpacks the addon to a specific folder), but I think it's not so easy for you - so I prepared the change: https://1284-215957316-gh.circle-artifacts.com/0/results/dist/blendnet-0.3.8-85e33b8.zip - could you please reinstall BlendNet from this archive and try to run again?

**Comment by lezan** [comment]
Hi @rabits thanks to support me.

I will try to follow what you say soon, now I am not able to try it.

> Oh, no worries about the node buttons - they are available only when instance can be actually created.

Oh ok, I had thought it!

> I mean more Addon logs, which is available when you run blender in console - they are printing to the stdout.

Ok, I will put all things. I thought they are not relevant.

>  but I think it's not so easy for you

Acutally I can :) but it was just to be sure!

So:
1. First all I will try to run another time with 0.3.7 and I will put logs here.
2. Then I will try with 0.3.8 and I will say the outcome.

Thanks!

**Comment by rabits** [comment]
Sure, take your time - but it will be better to switch the step order - first check the 0.3.8-85e33b8 dev version I prepared and in case it will not work - send the logs from it. There is no point now to send the logs from the old version, and I don't want you to waste your time :)

**Comment by lezan** [comment]
Hi @rabits,
With the new version and I am able to start the render (still rendering, I hope it will a success). If you need I can attach logs, if they can help you in some way.

I wanna ask you two questions:
1. Can I start a render with just one agent?
2. Using google console (without BlendNet) I created just one VM for render my blend file, for example e2-standard-16 machine. Using BlendNet instead it create two VM, one for the manager and a second one for the agent: machine assign to the manager need to be performant as agent, or I can just assign to it something not good like agent?

Thanks for you support!

**Comment by rabits** [comment]
Great it's working for you, so I will merge the fix and release the new version together with another one issue: https://github.com/state-of-the-art/BlendNet/issues/139

Regarding the questions:
1. For sure you can start render with one agent. This will bring a number of stability issues if you use cheap instances (that could be destroyed by GCP at any time), but overall there is no particular "can't" if you want to.
2. General idea of the Manager - is to distribute the tasks for "workhorses" Agents and make sure we have enough horse power in the provided limit of maximum instances, so the Manager could be small (to not waste your money) and Agents big (to be used when there is tasks to work on). For BlendNet v0.3 Manager also merges the samples into a complete picture, so uses some CPU power, but in v0.4 this operation is planned (https://github.com/state-of-the-art/BlendNet/issues/65) to be moved to the Agent.
So keep your Manager small and Agents somehow big (but not too big if you use cheap instances - they will be alive for too short period of time to be useful).

Thank you

**Comment by rabits** [comment]
@lezan I just released [v0.3.8](https://github.com/state-of-the-art/BlendNet/releases/tag/v0.3.8) with no intel optimization - so hopefully it's working well for you)
